src/get-started/flutter-for/react-native-devs.md

```diff
@@ -2593,7 +2593,7 @@ and common widget properties.
 [`Axis`]: {{site.api}}/flutter/painting/Axis.html
 [`BuildContext`]: {{site.api}}/flutter/widgets/BuildContext-class.html
 [`Center`]: {{site.api}}/flutter/widgets/Center-class.html
-[color palette]: {{site.material}}/design/color
+[color palette]: {{site.material}}/styles/color
```

Comment:
This should point to the "2014 Material Design color palettes" section under https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors